includes/admin/class-wp-job-manager-permalink-settings.php

```diff
@@ -118,12 +136,23 @@ public function settings_save() {
 				switch_to_locale( get_locale() );
 			}
 
-			$permalinks                  = (array) get_option( 'wpjm_permalinks', array() );
-			$permalinks['job_base']      = sanitize_title_with_dashes( $_POST['wpjm_job_base_slug'] );
-			$permalinks['category_base'] = sanitize_title_with_dashes( $_POST['wpjm_job_category_slug'] );
-			$permalinks['type_base']     = sanitize_title_with_dashes( $_POST['wpjm_job_type_slug'] );
+			/**
+			 * Option `wpjm_permalink` was renamed to match other options in 1.32.0.
```

Comment:
`wpjm_permalink` is missing the 's'.

Reply:
Fixed in a5980321f66801dfc5d6e0805cd31f4edf15f2d1